### PR TITLE
Feature/newplaybook

### PR DESCRIPTION
**Pull request by PizzaLi**
add task timeout controller

### CODE REVIEW

File: ansible/roles/kubefate/tasks/main.yml [hunk 1]
```diff
@@ -101,15 +101,30 @@
...
             - name: Running deployment script.
+              async: "{{ fate_create_timeout }}"
+              poll: "{{ check_interval }}"
               shell: "{{ base_dir }}/deploy.sh >> /tmp/cicd-{{ ansible_host }}.out"
-              register: done
+              register: fate_status
+              ignore_errors: true
+
+            - name: Clean
+              shell: /tmp/clean.sh
```
Comment (jiahaoc1993) [inline ansible/roles/kubefate/tasks/main.yml]: please replace "/tmp/" with "{{ base_dir }}/"

Comment (jiahaoc1993) [reply ansible/roles/kubefate/tasks/main.yml]: same as line 111

File: ansible/roles/kubefate/tasks/main.yml [hunk 2]
```diff
@@ -101,15 +101,30 @@
           register: exists
 
         - name: Create k8s cluster
+          async: "{{ kube_create_timeout }}"
+          poll: "{{ check_interval }}"
           shell: "{{ base_dir }}/utils/create_k8s.sh >> /tmp/cicd-{{ ansible_host }}.out"
+          register: kube_status
```
Comment (jiahaoc1993) [inline ansible/roles/kubefate/tasks/main.yml]: please replace "kube_status" with "k8s_status"

Comment (PizzaLi) [reply ansible/roles/kubefate/tasks/main.yml]: done

File: ansible/roles/kubefate/tasks/main.yml (hunk 1 above)
Comment (PizzaLi) [reply ansible/roles/kubefate/tasks/main.yml]: done

**Review by jiahaoc1993 — APPROVED** [review]

**Pull request merged**